criu/seize.c

```diff
@@ -542,7 +542,7 @@ static int freeze_processes(void)
 	enum freezer_state state = THAWED;
 
 	static const unsigned long step_ms = 100;
-	unsigned long nr_attempts = (opts.timeout * 1000000) / step_ms;
+	unsigned long nr_attempts = (opts.timeout * 1000) / step_ms;
```

Comment:
@kolyshkin Could you please add a comment here to explain why we are multiplying the timeout by 1000?

Reply:
I think it's clear (s to ms conversion), but will add.